docs/pages/access-controls/reference.mdx

```diff
@@ -418,14 +320,253 @@ spec:
       where: '!contains(ssh_session.participants, user.metadata.name)'
 ```
 
-## Second Factor - U2F
+## Role predicates
 
-Refer to the [Second Factor - WebAuthn](./guides/webauthn.mdx#u2f) guide if you
-have a cluster using the legacy U2F support.
+In a Teleport role resource, certain fields allow you to use functions and
+variables to configure a user's access. The functions and variables available
+for a field depend on the access controls that the field configures.
+
+### Access Request predicate functions
+
+The following variables and functions allow more fine-grained control over a
+user's permissions to submit and review Access Requests.
+
+#### Fields
+
+You can use Access Request predicate functions in the following fields. You can
+include these fields within either `allow` rules or `deny` rules:
+
+- `request.search_as_roles`
+- `request.roles`
+- `review_requests.preview_as_roles`
+- `review_requests.roles`
+- `review_requests.where`
+
+#### Functions
+
+Access Requests support the following role predicate functions:
+
+|Function|Description|
+|---|---|
+|`{{regexp.match(regexp)}}`|Returns `true` if the regular expression matches a user's role.|
+|`{{regexp.not_match(regexp)}}`|Returns `true` if the regular expression does not match a user's role.|
+
+Regular expressions support both wildcards (the `*` character) and [Go-style
+regular expressions](https://pkg.go.dev/regexp). If an expression begins with
+the `^` character and ends with the `$` character, Teleport will evaluate it as
+a regular expression. Otherwise, it will evaluate it as a wildcard expression.
+Wildcards match a sequence of one or more characters.
+
+#### Variables
+
+Teleport does not perform variable expansion on the values of the fields above.
+
+### Label expressions
+
+<Admonition type="warning">
```

Comment:
```suggestion
```
Does this really warrant a Warning?

Reply:
Because it's so new, I would say yes. (at least for the v13 and v12 versions of the docs)

If you don't see this, and you create a label expression on too old of a version, you will think you are enforcing security properties when you are not. This could lead to unintended infrastructure access, which would be a critical incident.